tools/armature.py

```diff
@@ -360,7 +360,7 @@ def execute(self, context):
                     coll = bpy.data.collections.get(name)
                     if not coll:
                         coll = bpy.data.collections.new(name)
-                        scene.collection.children.link(coll)
+                        bpy.context.scene.collection.children.link(coll)
```

Comment:
Use of bpy.context should be avoided when a function already has access to a context, because the function's context may be completely different to bpy.context, particularly if the operator has been called with a context override.

Reply:
Thanks for the feedback @Mysteryem ! I have rebased my changes to the current development branch, and used the passed context instead of the global. Does everything look OK?